chain/chain-site-rebuild.yml

```diff
@@ -3,6 +3,14 @@ command:
   name: site:rebuild
   description: 'Performs multiple commands to build a site (without git checkout)'
 commands:
+# Clear php opcode cache
+  - command: exec
+    arguments:
+      bin: 'sudo service php-fpm restart'
+# Clear memcache
```

Comment:
restart command doesn't work on my VM for some reason
to see if memcache is running do `ps aux | grep memcache`